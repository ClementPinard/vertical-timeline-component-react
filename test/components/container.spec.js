import React from 'react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';

import Container from '../../src/components/container';

Enzyme.configure({ adapter: new Adapter() });

describe('<Container />', () => {
	it('should create Snapshot', () => {
		const component = renderer.create(
			<Container>
				<div>First child</div>
				<div>Second child</div>
			</Container>,
		);

		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
