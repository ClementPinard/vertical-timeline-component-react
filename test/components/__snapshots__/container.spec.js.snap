// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should create Snapshot 1`] = `
<section
  className="sc-AxiKw eozVxv"
>
  <div>
    First child
  </div>
  <div
    className="sc-AxhCb ccMvAG"
  >
    <div>
      Second child
    </div>
  </div>
</section>
`;
